### PR TITLE
[terra-core-docs] terra-icon doc corrections

### PR DESCRIPTION
**Pull request by sdadn**
### Summary

Some corrections for `terra-icon` docs.

### Testing

This change was tested using:

- [ ] WDIO
- [ ] Jest
- [ ] Visual testing (please attach a screenshot or recording)
- [ ] Other (please describe below)
- [ ] No tests are needed

### Reviews

In addition to engineering reviews, this PR needs:

- [ ] UX review
- [ ] Accessibility review
- [ ] Functional review

### Additional Details

**This PR resolves:**

UXPLATFORM-XXXX 

---

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

**Review by trandrew1023 — APPROVED** [review]

**Review by adoroshk — APPROVED** [review]

File: packages/terra-core-docs/src/terra-dev-site/doc/icon/AccessibilityGuide.2.doc.mdx [hunk 1]
```diff
@@ -79,14 +79,14 @@ You should pick an `a11yLabel` that conveys the same information that a user vie
     <A11yMeaningfulReadOnly />
 </div>
 <div aria-label="Example code">
-
+```js
```
Comment (adoroshk) [inline packages/terra-core-docs/src/terra-dev-site/doc/icon/AccessibilityGuide.2.doc.mdx:82]: I guess here something went wrong with formatting as it looks like this (see highlited):
Screenshot 2023-09-20 at 4 57 29 PM

Comment (sdadn) [reply packages/terra-core-docs/src/terra-dev-site/doc/icon/AccessibilityGuide.2.doc.mdx:82]: This is an issue that already exists in main:

CleanShot 2023-09-20 at 17 50 01

The important tag is being misused here and the document needs to be rewritten, which while likely happen as part of the documentation uplift project.

CC: @mjpalazzo , something to take note of that will need fixing.

**Review by adoroshk — COMMENTED** [review]
Screenshot 2023-09-21 at 11 10 09 AM
 There is also a formatting problem below where the documentation is placed inside the code example where it should not.

File: packages/terra-core-docs/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Fixed
+  * `terra-icon` documentation corrections.
```
Comment (nikhitasharma) [inline packages/terra-core-docs/CHANGELOG.md:6]: nit: Can we specify that these are import corrections. Just corrections seems unclear as to what was changed/updated.

**Review by adoroshk — APPROVED** [review]

File: packages/terra-core-docs/CHANGELOG.md (hunk 2 above)
Comment (sdadn) [reply packages/terra-core-docs/CHANGELOG.md]: Fixed here: 7dbae3d2047fd184887a6a46bb9b3637318183cd

File: packages/terra-core-docs/src/terra-dev-site/doc/icon/AccessibilityGuide.2.doc.mdx (hunk 1 above)
Comment (sdadn) [reply packages/terra-core-docs/src/terra-dev-site/doc/icon/AccessibilityGuide.2.doc.mdx]: Correction, this was resolved in: 4533e1b000d7800c324494d791e8eedce0463125

**Review by adoroshk — APPROVED** [review]
Reviewed the changes, LGTM
Screenshot 2023-09-21 at 2 31 10 PM

**Pull request merged**